pdctl/command/namespace_command.go

```diff
@@ -23,18 +23,19 @@ import (
 
 const (
 	namespacePrefix    = "pd/api/v1/namespaces"
-	namespaceAddPrefix = "pd/api/v1/namespaces/add"
+	namespaceRelPrefix = "pd/api/v1/namespaces/rel"
```

Comment:
I am confused what dos `rel` mean here?

Reply:
What I means here is relationship. @disksing raised the same question, maybe we need a new name to make it more clear.

Reply:
I think you can use `/namespaces/table` here.